hazelcast/src/test/java/com/hazelcast/client/console/ClientConsoleTest.java

```diff
@@ -71,7 +76,7 @@ public void after() {
      * YAML client configuration file.
      */
     @Test
-    public void connectsToHazelcastCluster() throws IOException {
+    public void connectsToHazelcastCluster() throws IOException, InterruptedException {
```

Comment:
Throwing `InterruptedException` is not needed here.

Reply:
:+1: Nice spot. I'll fix it. It was introduced by playing with a `Thread.sleep` within the test.